src/ledger/address.ts

```diff
@@ -39,6 +39,11 @@ export class Address implements ABIValue {
 	isZero(): boolean {
 		return this.value.every((x) => x === 0);
 	}
+
+	equals(other: Address): boolean {
+		return (this.value.length === other.value.length)
```

Comment:
`( ... )` are not needed here.

Up to you if you leave them or not, I just wanted to comment something here because I like the use of `.every(...)` :D

Reply:
I prefer to leave as is because it separates more clearly from the next boolean expression.